src/App.js

```diff
@@ -1,12 +1,16 @@
 import React, { useEffect } from 'react';
 import { Provider } from 'react-redux';
 import { sessionService } from 'redux-react-native-session';
+import api from 'api';
+import applyDefaultInterceptors from 'api/utils/applyDefaultInterceptors';
 
 import AppContainer from './navigators';
 import configureStore from './store/configureStore';
 
 const store = configureStore({});
 
+applyDefaultInterceptors(api);
```

Comment:
I don't quite get this default interceptors concept, why should I create Interceptors outside this module? seems to me that all the Interceptors will be defined inside this module; maybe I'm missing something.